rdt/transformers/__init__.py

```diff
@@ -1,5 +1,8 @@
 """Transformers module."""
 
+from collections import defaultdict
+from functools import lru_cache
```

Comment:
I'm not sure if `functools` is part of the standard library. If it is not, we should add it to `setup.py`, even if we already install it because one of our dependencies use it, so if they remove it in the future we do not crash.

Reply:
It is part of the standard library